### DISCUSSION TITLE
recent change "feat: prefer deno.json imports over deps.ts"

### DISCUSSION THREAD

**Discussion by deer** [oliver-oloughlin/kvdex#179] [General]
Hi @oliver-oloughlin, why was https://github.com/oliver-oloughlin/kvdex/commit/dbc89dc8d30a4aaaecea5da5f56d933918e5de6e made? I don't understand how I'm easily supposed to use this now. (Although I must admit this is my first time using kvdex...)

I'm trying to create a simple CLI that makes use of deno kv for storing information across invocations. And when the user installs the CLI, there's no import map. When I try to run my program, I get a bunch of errors about missing imports. From my understanding, the general advice is that libraries/frameworks should use `deps.ts` and apps should use import maps. Can you help me understand your reasoning and how to proceed?

For now I've just downgraded to `0.34.1` and everything works well so far. Thanks for the cool module!

**Comment by oliver-oloughlin** [comment]
Hi, sorry for the misunderstanding, I should have made this clear in the chnagelogs:
I'm moving kvdex to jsr, and for jsr import maps are advised over deps.ts. However, that means it breaks the library when accessing it from deno.land/x. I meant to remove the webhook that publishes to deno.land/x, but forgot for this release, hence why the latest verison on deno.land/x is broken.

Going forward, new releases will ony be published to jsr.io. I therefore recommend using the jsr imports )and updating to the latest verison of Deno. You can find examples of how jsr imports look in the README.

**Comment by deer** [comment]
Perfect. It's all sorted now -- thanks!
